tests/unit/Entity/ItemIdSetTest.php

```diff
@@ -16,6 +16,35 @@
  */
 class ItemIdSetTest extends \PHPUnit_Framework_TestCase {
 
+	public function testGetIterator() {
+		$set = new ItemIdSet();
+		$iterator = $set->getIterator();
+
+		$this->assertInstanceOf( 'Traversable', $iterator );
+	}
+
+	/**
+	 * @dataProvider serializationsProvider
+	 */
+	public function testGetSerializations( array $expected ) {
```

Comment:
I'm still not sure how this should look like instead, in your opinion. Like I did it now?